### PR TITLE
Add support for default value of Inclusive minPt parameter, if not se…

### PR DESCRIPTION
**Pull request by lstroem**
BEGINRELEASENOTES
- FastJetUtil/FastJetProcessor: Print error message if value of Inclusive minPt parameter is not specified in the steering file.
  - set default value minPt="0.0"

ENDRELEASENOTES

### CODE REVIEW

File: include/FastJetUtil.h [hunk 1]
```diff
@@ -430,7 +431,7 @@ void FastJetUtil::initClusterMode() {
 
   // check the different cluster mode possibilities, and check if the number of parameters are correct
   if (_clusterModeName.compare("Inclusive") == 0) {
-
+    if ((_clusterModeNameAndParam.size() == 1) && (_clusterModeNameAndParam[0] == "Inclusive")) { _clusterModeNameAndParam.push_back("0."); }
```
Comment (andresailer) [inline include/FastJetUtil.h]: Can you add a message here that the default value is being used, please.

**Review by gaede — COMMENTED** [review]
Looking at the code, I think that the old behavior is actually better, as it requires the user to provide in the steering file a value for the mode and the ptmin parameter - otherwise an exception with the correct error message is thrown.
Why do you want to rely on "hard coded" default parameters that are not shown with the usual Marlin parameter handling ?

**Comment by lstroem** [comment]
André: I added a message saying that the default value is used if minPt is not configured.
Frank: If that is what is preferred we should at least catch the error, right now it gives me the output pasted below:

A runtime error occured - (uncaught exception):
      lcio::Exception: Wrong Parameter(s) for Clustering Mode. Expected:
 \<parameter name="clusteringMode" type="StringVec"\> Inclusive \<minPt\> \</parameter\>"
 Marlin will have to be terminated, sorry.
 ***********************************************

Or maybe this "uncaught exception" is normal?

**Comment by gaede** [comment]
@lstroem in principle this uncaught exception is the right thing, as the steering file is incomplete/incorrect. To make it yet more clear to the users you could add a printout right before throwing the exception, e.g.:
```
streamlog_out(ERROR)  << "Wrong numbers values for parameter clusteringMode "Inclusive" : missing minPt" << std::endl
```
this way the user immediately knows which processor to update.
Note this is done for other parameters in this code.

**Comment by lstroem** [comment]
ok, thanks for the comments! New version pushed to fork feature branch.
I've implemented the new comment suggested instead of the default stuff. Indeed it is more clear than before, since this error shows up in the bottom of the output log file instead of hidden in the bulk text somewhere.

**Review by gaede — APPROVED** [review]

**Pull request merged**